### PR TITLE
feat(staking): add feature flag for new grid view

### PR DESCRIPTION
**Pull request by przemyslaw-wlodek**
# Checklist

- [x] [JIRA](https://input-output.atlassian.net/browse/LW-9266)

---

## Proposed solution
Add a new feature flag for staking that will enable switching between Grid and List view when browsing pools.

## Testing
To be covered in the other PR (Grid View).

### CODE REVIEW

File: apps/browser-extension-wallet/.env.example [hunk 1]
```diff
@@ -28,6 +28,7 @@ USE_POSTHOG_ANALYTICS_FOR_OPTED_OUT=false
 USE_MATOMO_ANALYTICS_FOR_OPTED_OUT=false
 USE_COMBINED_PASSWORD_NAME_STEP_COMPONENT=false
 USE_MULTI_DELEGATION_STAKING_ACTIVITY=true
+USE_MULTI_DELEGATION_STAKING_GRID_VIEW=false
```
Comment (vetalcore) [inline apps/browser-extension-wallet/.env.example:31]:
```suggestion
USE_MULTI_DELEGATION_GRID_VIEW=false
```
wdyt?

**Review by vetalcore — CHANGES_REQUESTED** [review]

File: apps/browser-extension-wallet/.env.example (hunk 1 above)
Comment (przemyslaw-wlodek) [reply apps/browser-extension-wallet/.env.example:31]: I'd say yes but I want to be consistent with all variables we have. 😄 I think it'd be worth renaming after we drop the old single-staking impl.
```
USE_MULTI_DELEGATION_STAKING_LEDGER=false
USE_MULTI_DELEGATION_STAKING_TREZOR=false
...
USE_MULTI_DELEGATION_STAKING_ACTIVITY=true
```

Comment (vetalcore) [reply apps/browser-extension-wallet/.env.example:31]: could you please leave a todo then?

**Review by vetalcore — APPROVED** [review]

File: apps/browser-extension-wallet/.env.example (hunk 1 above)
Comment (przemyslaw-wlodek) [reply apps/browser-extension-wallet/.env.example:31]: Done :)

**Review by mchappell — APPROVED** [review]

**Pull request merged**